### PR TITLE
Add release.yaml for v0.13.2

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

**Comment by chmouel** [comment]
/lgtm
/approve

🙇

File: openshift/release/tektoncd-pipeline-v0.13.2.yaml [hunk 1]
```diff
@@ -0,0 +1,1697 @@
...
+          "-git-image", "quay.io/openshift-pipeline/tektoncd-pipeline-git-init:v0.13.2",
+          "-entrypoint-image", "quay.io/openshift-pipeline/tektoncd-pipeline-entrypoint:v0.13.2",
+          "-imagedigest-exporter-image", "quay.io/openshift-pipeline/tektoncd-pipeline-imagedigestexporter:v0.13.2",
+          "-pr-image", "quay.io/openshift-pipeline/tektoncd-pipeline-pullrequest-init:v0.13.2",
+          "-build-gcs-fetcher-image", "quay.io/openshift-pipeline/tektoncd-pipeline-gcs-fetcher:v0.13.2",
+
+          # This image is used as a placeholder pod, the Affinity Assistant
+          # TODO(#2640) We may want to create a custom, minimal binary
+          # As of June 8, 2020, tag 1.19.0
+          "-affinity-assistant-image", "nginx@sha256:c870bf53de0357813af37b9500cb1c2ff9fb4c00120d5fe1d75c21591293c34d",
```
Comment (chmouel) [inline openshift/release/tektoncd-pipeline-v0.13.2.yaml:1436]: We really shoudl prioritize to fix that (be it upstream of if that takes too long, downstream)

Comment (vdemeester) [reply openshift/release/tektoncd-pipeline-v0.13.2.yaml:1436]: @chmouel what do you mean, this is "kinda" fix, it won't fail any more at least.

**Pull request merged**